_scripts/update.py

```diff
@@ -145,7 +150,7 @@ def update_str(
     with open(data_path, "w") as file:
         json.dump(messages, file, indent=2)
 
-    return new_files, updated_files
+    return new_files, updated_files, paths
```

Comment:
The paths need to be a sorted list for the comparison.

```suggestion
    return new_files, updated_files, sorted(list(paths))
```